lib/connection/result/data_types.js

```diff
@@ -221,6 +221,16 @@ const nativeTypes =
     }
   };
 
+let isFetchOnlyNull = false;
```

Comment:
should we perhaps rename this, then  227 and 231 too, with the change of the flag name ?

Reply:
I replaced them.

Reply:
thank you !